lib/router.js

```diff
@@ -84,6 +84,7 @@ router.get('/accounts/:account/orders', api.orders.getOrders);
 router.post('/accounts/:account/orders', api.orders.placeOrder);
 router.delete('/accounts/:account/orders/:sequence', api.orders.cancelOrder);
 router.get('/accounts/:account/order_book/:base/:counter', api.orders.getOrderBook);
+router.get('/accounts/:account/order_changes/:identifier', api.orders.getOrderChange);
```

Comment:
I would call it hash instead of identifier. In the case of payments it can be a hash or client_resource_id, but here it's always a hash